src/main/modules/i18n/index.ts

```diff
@@ -3,7 +3,7 @@ import * as middleware from 'i18next-express-middleware'
 import * as express from 'express'
 
 import i18next from 'i18next'
-import { Backend } from './backend'
+const Backend = require('i18next-node-fs-backend')
```

Comment:
just to keep it consistent, shall we do this - `import Backend from 'i18next-node-fs-backend'`

you have to `yarn add @types/i18next-node-fs-backend` for import to work.